content/react.md

```diff
@@ -0,0 +1,271 @@
+---
+title: React
+order: 8
+description: How to use React, the frontend rendering library with Meteor.
+---
+
+After reading this guide, you'll know:
+
+1. What React is, and why you would consider using it with Meteor.
```

Comment:
Maybe backwards? Like "Why you should consider using Meteor as a backend for a React app"? Not sure what the target audience is TBH